histomicstk/segmentation/nuclear/detect_nuclei_kofahi.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+import scipy as sp
+import skimage.morphology
+import histomicstk.filters.shape as htk_shape_filters
+import histomicstk as htk
+
+
+def detect_nuclei_kofahi(im_nuclei_stain, foreground_threshold, min_radius,
+                         max_radius, min_nucleus_area, local_max_search_radius):
+
+    """Performs a nuclear segmentation using kofahi's method.
+
+    Parameters
+    ----------
+    im_nuclei_stain : array_like
+        An intensity image used  to extract the image foreground using a
+        graph-cuts-based binarization and to detect nuclear seed points which
+         are used to perform an initial segmentation that is refined using a
+        second graph-cuts-based algorithm. Dimensions M x N.
+    foreground_threshold : float
+    min_radius : float
+    max_radius : float
+    min_nucleus_area : int
+    local_max_search_radius : float
```

Comment:
Can you add a description to all these parameters? You can find them either in the NucleiDetection XML or within other functions it calls withing this function.